packages/components/src/internal/components/picklist/ChoosePicklistModal.tsx

```diff
@@ -227,33 +229,63 @@ export const ChoosePicklistModalDisplay: FC<ChoosePicklistModalProps & ChoosePic
             onCancel,
             afterAddToPicklist,
             user,
-            selectionKey,
+            selectionKey: initSelectionKey,
             numSelected,
-            sampleIds,
+            sampleIds: initSampleIds,
```

Comment:
I see, so this is a renaming of the incoming prop so that you can reuse that variable name as a state variable. Yeah, I think this is a good time to use the _ prefix syntax or to change the incoming prop to `initSampleIds` to be clear on how / where they are being used.

Reply:
another option would be to do the selection --> sampleId higher up the stack, and then use the existing sampleId parameter.